core/templates/dev/head/pages/dashboard/Dashboard.js

```diff
@@ -92,6 +92,26 @@ oppia.controller('Dashboard', [
     $scope.unresolvedAnswersIconUrl = UrlInterpolationService.getStaticImageUrl(
       '/icons/unresolved_answers.svg');
 
+    // Refreshes the dashboard page whenever the back button is pressed
+    $scope.$watch(function() {
+      return $location.path();
+    }, function(newPath, oldPath) {
+      if (performance.navigation.type !== 1) {
+        if (newPath === '' && oldPath === '') {
+          $location.path('').replace;
```

Comment:
You know, I'm a bit confused by this construct. Is replace() a function? If not, does this line actually do anything?

Appreciate any links to docs, if you have them. I looked through https://docs.angularjs.org/api/ng/service/$location and it seems to indicate "replace()" but apparently it works fine without the parentheses (based on Arunabh's report of everything working fine)?